plot/plot.app.js

```diff
@@ -1292,15 +1292,12 @@
                                                     }
                                                 }),
                                                 element[0].on('plotly_legendclick', function(data){
-                                                    var contextUrlParams=ConfigUtils.getContextHeaderParams();
+                                                    // var contextUrlParams=ConfigUtils.getContextHeaderParams();
```

Comment:
Post today's discussion, I realized the I had written redundant code for adding the markdown pattern links,
Removed it in this PR